### PR TITLE
AT_04.13.01 | Create booking page > Departure date > Month dropdown menu (to the left of the Week button) has 13 months for selection

### PR DESCRIPTION
**Pull request by SvetlanaKanel**
https://trello.com/c/KEsuUyfG/234-at041301-create-booking-page-departure-date-month-dropdown-menu-to-the-left-of-the-week-button-is-clickable-and-has-13-months-fo

### CODE REVIEW

**Review by LIliia149 — APPROVED** [review]

File: cypress/e2e/testUiAndFunction/createBookingPage/departureDateSection/US_04.13_MonthDropdownMenu.cy.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/// <reference types = "Cypress" />
+
+import CreateBookingPage from "../../../../pageObjects/CreateBookingPage.js";
+
+const createBookingPage = new CreateBookingPage();
+
+describe('US_04.13 | Create booking page > Departure date > Month dropdown UI and functionality', () => {
```
Comment (kgoldpasv) [inline cypress/e2e/testUiAndFunction/createBookingPage/departureDateSection/US_04.13_MonthDropdownMenu.cy.js:7]: can you please delete this: Create booking page > Departure date- extra info

File: cypress/e2e/testUiAndFunction/createBookingPage/departureDateSection/US_04.13_MonthDropdownMenu.cy.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    beforeEach(function () {
+        cy.visit('/');
+        cy.login(AGENT.email, AGENT.password);
+        createBookingPage.clickMonthBtn();
+    })
+
+    it('AT_04.13.01 | Month dropdown menu (to the left of the Week button) is visible and has 13 months for selection', () => {
+        createBookingPage.clickMonthDropdown();
+        createBookingPage.getMonthDropdownList().should('be.visible')
```
Comment (kgoldpasv) [inline cypress/e2e/testUiAndFunction/createBookingPage/departureDateSection/US_04.13_MonthDropdownMenu.cy.js:18]: can you put this . to separate lines:like this
.getMonthDropdownList()
.should('be.visible')

File: cypress/pageObjects/CreateBookingPage.js [hunk 3]
```diff
@@ -22,5 +25,16 @@ class CreateBookingPage {
...
+        this.getMonthBtn().click({ forse: true });
+    }
+
+    selectMonthDropdownList() {
+        this.getMonthDropdownList().select(0);
+    }
+
+    clickMonthDropdown() {
+        this.getMonthDropdown();
+    }
```
Comment (kgoldpasv) [inline cypress/pageObjects/CreateBookingPage.js:38]: this method did not do anything, why did you add it?

Comment (SvetlanaKanel) [reply cypress/pageObjects/CreateBookingPage.js:38]: A user has to click on the Month dropdown menu to see the list of the months, and after that the user can select a month

File: cypress/e2e/testUiAndFunction/createBookingPage/departureDateSection/US_04.13_MonthDropdownMenu.cy.js (hunk 2 above)
Comment (SvetlanaKanel) [reply cypress/e2e/testUiAndFunction/createBookingPage/departureDateSection/US_04.13_MonthDropdownMenu.cy.js:18]: could you please specify what do you mean?
